### ISSUE TITLE
$cluster.DrsAutomationLevel pulls in as an int

### ISSUE THREAD

**Issue by chriswahl** [WahlNetwork/Vester#65]
For some interesting reason, the JSON file created by `New-VesterConfig` is generating an int value for drsmode (based on $cluster.DrsAutomationLevel). The value being pulled from vCenter is definitely `FullyAutomated` so I'm guessing the `ConvertTo-Json` cmdlet is to blame.

Can anyone else validate this behavior? Here's a snipet from my JSON config.

```
"cluster":  {
                    "drsmode":  1,
                    "drslevel":  3,
                    "haenable":  true
```

/cc @brianbunke @equelin

**Comment by brianbunke** [comment]
Did replicate. This one is pretty lol.

    PS C:\> (Get-Cluster).DrsAutomationLevel.GetType()

    IsPublic IsSerial Name                                     BaseType                                                                                                      
    -------- -------- ----                                     --------                                                                                                      
    True     True     DrsAutomationLevel                       System.Enum

Fixed in commit 09714b1.

**Comment by chriswahl** [comment]
Nice catch. :)
